keyboards/bemeier/bmek/rules.mk

```diff
@@ -0,0 +1,23 @@
+MCU = atmega32u4
+BOOTLOADER = atmel-dfu
+BOOTMAGIC_ENABLE = lite
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = yes
+CONSOLE_ENABLE = no
+COMMAND_ENABLE = no
+SLEEP_LED_ENABLE = yes
+SPLIT_KEYBOARD = no
+BACKLIGHT_ENABLE = no
+NKRO_ENABLE = yes
+RGBLIGHT_ENABLE = yes
+UNICODE_ENABLE = no
+UNICODEMAP_ENABLE = no
+BLUETOOTH_ENABLE = no
+AUDIO_ENABLE = no
+FAUXCLICKY_ENABLE = no
+DYNAMIC_MACRO_ENABLE = yes 
+DEFAULT_FOLDER = bemeier/bmek/rev3
+LEADER_ENABLE = no
+KEY_LOCK_ENABLE = no
+TERMINAL_ENABLE = no
+LINK_TIME_OPTIMIZATION_ENABLE = yes
```

Comment:
```suggestion
# MCU name
MCU = atmega32u4

# Bootloader selection
BOOTLOADER = atmel-dfu

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE      = lite
MOUSEKEY_ENABLE       = no
EXTRAKEY_ENABLE       = yes
CONSOLE_ENABLE        = no
COMMAND_ENABLE        = no
SLEEP_LED_ENABLE      = yes
SPLIT_KEYBOARD        = no
BACKLIGHT_ENABLE      = no
NKRO_ENABLE           = yes
RGBLIGHT_ENABLE       = yes
AUDIO_ENABLE          = no
DYNAMIC_MACRO_ENABLE  = no
LTO_ENABLE            = yes

DEFAULT_FOLDER = bemeier/bmek/rev3
```

Also, I'm not sure that you want `DYNAMIC_MACRO_ENABLE` set here, as it is independant of what VIA does.